docs/usage.rst

```diff
@@ -54,7 +54,7 @@ Most ISIS programs have a ``FROM=`` parameter, and so all kalasiris
 versions of them will assume that the first item in the argument
 signature is what should be assigned to the ``FROM=`` parameter if
 you were typing the ISIS program at the command line, that's one
-of the reasons why in the above aren't written ``isis.catlab(from=from_cub,
+of the reasons why in the above examples aren't written like ``isis.catlab(from=from_cub,
```

Comment:
Yes, this was poor of me.  I think it would read even better without the "in" here.  Thanks.